third-party/linshare/src/test/java/org/apache/james/linshare/LinshareTest.java

```diff
@@ -58,4 +64,17 @@ void linshareShouldStart() {
         .then()
             .statusCode(HttpStatus.SC_OK);
     }
+
+    @Test
+    void linshareShouldShouldHaveATechnicalAccountConfigured() {
+        List<TechnicalAccountDTO> technicalAccounts = linshareExtension.getAllTechnicalAccounts(LinshareFixture.ADMIN_ACCOUNT);
+
+        Predicate<TechnicalAccountDTO> technicalAccountCondition = technicalAccount ->
+            technicalAccount.getName().equals(TECHNICAL_ACCOUNT.getUsername())
+            && technicalAccount.getPermissions().equals(TECHNICAL_PERMISSIONS);
+
+        assertThat(technicalAccounts).matches(accounts ->
+            accounts.stream().anyMatch(technicalAccountCondition)
+        );
```

Comment:
this test is correct, but the style is little different than James convention.

```java
void linshareShouldCreateATechnicalAccount() {
    TechincalAccount technicalAccount = TechincalAccount.defaultAccount();
    linshareExtension.createTechnicalAccount(technicalAccount);

    assertThat(linshareExtension.getAllTechnicalAccounts())
       .containsOnly(technicalAccount);
}
```

In order to do that, there is a prerequisite: override equals and hashCode in  TechincalAccount and write a EqualVerifier test for it (org.apache.james.linshare.client.DocumentTest is an example)